src/pkg/caendr/caendr/models/run/gcp.py

```diff
@@ -346,7 +372,7 @@ def _log_giveup(details):
   @backoff.on_exception(
       backoff.constant, SSLEOFError, max_tries=_MAX_TRIES_TOTAL, interval=20, jitter=None, on_backoff=_log_ssl_backoff.__func__, on_giveup=_log_ssl_giveup.__func__,
   )
-  def run(self, report, run_if_exists=False):
+  def run(self, command: list, env: dict, container_uri: str, params: dict, run_if_exists: bool = False) -> str:
```

Comment:
The `Runner` now takes these as arguments, since they're constructed by `JobPipeline` using data from `Report`